data/formats-data.js

```diff
@@ -3713,7 +3720,7 @@ exports.BattleFormatsData = {
 		tier: "UU",
 	},
 	piplup: {
-		randomBattleMoves: ["stealthrock", "hydropump", "scald", "icebeam", "hiddenpowerelectric", "hiddenpowerfire", "yawn", "defog"],
+		randomBattleMoves: ["stealthrock", "scald", "icebeam", "icywind", "defog"],
```

Comment:
scald, ice beam, defog, sr, yawn

Reply:
the only moves id consider wanting yawn over on a piplup randoms set are those of the ice type
icy wind can go tho

Reply:
yeah i guess removing icy wind should be enough then